lib/modules/manager/mise/extract.ts

```diff
@@ -24,8 +41,22 @@ export function extractPackageFile(
   if (tools) {
     for (const [name, toolData] of Object.entries(tools)) {
       const version = parseVersion(toolData);
-      const depName = name.trim();
-      const toolConfig = getToolConfig(depName, version);
+      // Parse the tool options in the tool name
+      const { name: depName, options: optionsInName } =
+        // istanbul ignore next: groups is always defined
+        optionInToolNameRegex.exec(name.trim())?.groups ?? {
+          name: name.trim(),
+        };
```

Comment:
```suggestion
      const { name: depName, options: optionsInName } = optionInToolNameRegex.exec(name.trim())!.groups!;
```

it it always matches, then do this.